src/main/java/frc/robot/RobotContainer.java

```diff
@@ -4,15 +4,31 @@
 
 package frc.robot;
 
+import frc.robot.subsystems.LeftClimberSubsystem;
+import frc.robot.commands.Autos;
+
 //import frc.robot.Constants.OperatorConstants;
 
 import frc.robot.subsystems.IntakeMechanisum;
 import frc.robot.subsystems.LanuchMechanisumSubsystem;
+import frc.robot.subsystems.LimelightNavigation;
 import frc.robot.subsystems.MecanumDriveSubsystem;
+import frc.robot.subsystems.RightClimberSubsystem;
 import frc.robot.subsystems.TrapScoreSubsystem;
+import frc.robot.subsystems.climberSubsystem;
+
+import java.security.cert.TrustAnchor;
```

Comment:
Looks like another unused import.

Reply:
witch one?

Reply:
Fr tho, which one?

Reply:
`import java.security.cert.TrustAnchor;`

Reply:
I will remove all unused imports after testing in case we forgot to add something back after removing.

Reply:
added it to the task list